### DISCUSSION TITLE
GitHub 用户专属福利，实际到账 3K+，Namebase Airdrop — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#236] [Announcements]
https://mazhuang.org/2020/02/21/namebase-airdrop/ 

 我经常提醒自己的防骗第一准则：天上不会掉馅饼。

**Comment by Zxilly** [comment]
14个follower的想跳楼

**Comment by mzlogin** [comment]
> 14个follower的想跳楼

哈哈，那14个是老粉吗？其实也可以试试，万一当时有人临时follow过后来又取消了呢
